core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -301,4 +301,5 @@ InvalidInputForXmlTransform=Invalid input for XMLTRANSFORM xml: ''{0}''
 InvalidInputForExtractValue=Invalid input for EXTRACTVALUE: xml: ''{0}'', xpath expression: ''{1}''
 InvalidInputForExtractXml=Invalid input for EXTRACT xpath: ''{0}'', namespace: ''{1}''
 InvalidInputForExistsNode=Invalid input for EXISTSNODE xpath: ''{0}'', namespace: ''{1}''
+DifferentLengthForBitwiseOperands=Different length for bitwise operands: the first: {0,number,#}, the second: {1,number,#}
```

Comment:
@wangxlong I am planning to get this into next release.
Can you add a negative test case to show this error message?
Thanks

Reply:
Also pelease rebase to the latest master.

Reply:
@hsyuan Yes, Done.